### ISSUE TITLE
Do we really need an Android app?

### ISSUE THREAD

**Issue by JJ** [jmv74211/TFM_security_system_PI#6]
I think it's more than enough if you create a progressive web app.

**Comment by jmv74211** [comment]
My main idea is to make a set of microservices on the server (in this case in the raspberry PI), and the user access interface would be using an application of your smartphone that communicates with the API of the server, either through a telegram bot or an android application.

First I'm going to start developing a telegram bot, and finally (if time allows, I hope so) I will create an android application to get and visualize the information.

**Comment by JJ** [comment]
@jmv74211 You can do a web application that runs on the phone, no need to do two different ones. Maybe with React Native.

**Comment by jmv74211** [comment]
Yeah, you're right. For reasons of time I could not contemplate the idea of developing the application using React Native. Maybe it can be implemented in future versions. Thanks for the advice.
